### PR TITLE
Add recipe

### PR DESCRIPTION
**Pull request by ozanozisik**
Addition of recipe for orsum, a tool for over-representation analysis result summary.
@BiocondaBot please update

### CODE REVIEW

File: recipes/orsum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  host:
+    - python >=3.6
+  run:
+    - python >=3.6
+    - argparse
+    - numpy >=1.18.0
+
+test:
+  commands:
+    - orsum -h
```
Comment (fanninpm) [inline recipes/orsum/meta.yaml:25]: See if this works:
```suggestion
    - orsum.py -h
```

Comment (fanninpm) [reply recipes/orsum/meta.yaml:25]: You also may need to add a build script that makes `orsum.py` executable.

For example, in a `build.sh` file, you could have:
```bash
#!/bin/bash

cp *.py $PREFIX/bin
chmod +x $PREFIX/bin/orsum.py
```

Comment (ozanozisik) [reply recipes/orsum/meta.yaml]: @fanninpm Thank you very much, - orsum.py -h and creating build.sh solved it.

**Comment by fanninpm** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**